src/components/common/PageCard.tsx

```diff
@@ -96,9 +118,11 @@ export default function PageCard(props: PageCardProps): JSX.Element {
         {description}
       </Typography>
       {linkStyle === 'plain' && (
-        <Link className={classes.link} component={RouterLink} to={link}>
-          {linkText}
-        </Link>
+        <Box onClick={stopBubblingEvent}>
```

Comment:
This is needed to prevent event from reaching up to the Box for double processing.
Also this is needed as a wrapper so we can preserver cmd+click behavior on the link.